eng/scripts/Show-FailureLogs.ps1

```diff
@@ -19,7 +19,11 @@ foreach ($logFile in $filteredLogs)
     Write-Host "=============================================================================================================================="
     Write-Host "Log file: $logFile"
     Write-Host "=============================================================================================================================="
-    Get-Content $logFile | Write-Host
+    try { 
+        Get-Content $logFile | Write-Host
```

Comment:
Why not do a Test-Path here instead of a try/catch?

Reply:
Also what is actually deleting the files in parallel here that would cause this to fail?

Reply:
If it's a race condition we could still fail when we `Get-Content`

Reply:
Do you have any idea what would delete the file while we are looping over them?

Reply:
After having a look my best guess is that an empty log file is still around when you query for logs but then once the full vcpkg process shuts down it removes that empty file. Given that this is just about dumping the log file if it exists I'm find with this solution.

Reply:
As far as I can tell, vcpkg should have exited by the time this script is running. 

I checked locally to see if the output of `Get-ChildItem` fed into `Get-Content` was somehow getting the wrong set of folders leading up to the file. That doesn't appear to be the case. I added more logging in the event that there's a factor on the build machine that's not being considered.